### PR TITLE
QSP-1189-Added short description to upgrade the node

### PR DESCRIPTION
**Pull request by martinetlee**
## Description

Adding description on how to upgrade the node

## Motivation and Context

Users don't need to follow the whole setup process for upgrade.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: doc/node-operator.md [hunk 1]
```diff
@@ -342,3 +342,12 @@ When you decide to stop operating a Quantstamp Protocol node, you might want to
     1. Navigate to the unstake function
     1. Click the `Write` Button
 
+### Upgrade your Node
+
+From time to time, you will need to upgrade the node to its latest version. 
+The following steps are all you need to upgrade your node. 
+As a node operator, you won't need to worry about to unstake from the old contract and stake into the new smart contract.
```
Comment (alex-murashkin) [inline doc/node-operator.md]: Let's add `Unless we do a major version change`

File: doc/node-operator.md [hunk 2]
```diff
@@ -342,3 +342,12 @@ When you decide to stop operating a Quantstamp Protocol node, you might want to
...
     1. Click the `Write` Button
 
+### Upgrade your Node
+
+From time to time, you will need to upgrade the node to its latest version. 
+The following steps are all you need to upgrade your node. 
+As a node operator, you won't need to worry about to unstake from the old contract and stake into the new smart contract. 
+
+1. [Download the bundle](#1-download-the-bundle)
+1. [Copy the `[MyEtherWallet keystore file]` into a specific location](#4-local-machine-setup)
```
Comment (alex-murashkin) [inline doc/node-operator.md]: Let's clarify that we need to extract the bundle into a new folder and move the keystore file

File: doc/node-operator.md (hunk 1 above)
Comment (martinetlee) [reply doc/node-operator.md]: Done!

File: doc/node-operator.md (hunk 2 above)
Comment (martinetlee) [reply doc/node-operator.md]: Thanks! Text updated, 

Also, I wanted to confirm with you: Are these two steps all that are required to upgrade the node? Making sure that I did not miss anything :P

File: doc/node-operator.md [hunk 3]
```diff
@@ -342,3 +342,13 @@ When you decide to stop operating a Quantstamp Protocol node, you might want to
...
 
+### Upgrade your Node
+
+From time to time, you will need to upgrade the node to its latest version. 
+The following steps are all you need to upgrade your node. 
+As a node operator, you won't need to worry about to unstake from the old contract and stake into the new smart contract unless we do a major version change. 
+
+1. [Download the bundle](#1-download-the-bundle) and extract it to a new folder
+1. Your keystore file in the original node directory, move the keystore file `resources/keystore/default.json` to the same location under the new folder. 
+
```
Comment (alex-murashkin) [inline doc/node-operator.md]: Let's make sure the old node has completely stopped (directing the operators to the Stop section). That would be step 3. And Step 4 should be running the node.
And step 5 - looking at the dashboard and confirming that your version was indeed incremented.

File: doc/node-operator.md (hunk 2 above)
Comment (alex-murashkin) [reply doc/node-operator.md]: Sure - added more comments

**Pull request merged**